fili-core/src/main/java/com/yahoo/bard/webservice/async/jobs/stores/NoOpApiJobStore.java

```diff
@@ -30,4 +32,10 @@
     public Observable<JobRow> getAllRows() {
         return Observable.empty();
     }
+
+    @Override
+    public Observable<JobRow> getFilteredRows(Set<JobRowFilter> jobRowFilter)
+            throws IllegalArgumentException {
```

Comment:
Can unwrap.

Also, we should probably document what this will return, since the NoOp version is a little odd, in general, and it may not be clear what filtering on the NoOp store would do.